front/src/common/Map/Consts/colors.ts

```diff
@@ -11,6 +11,8 @@ const speedOver220 = '#91d3ff';
 
 const colors: Record<string, Theme> = {
   normal: {
+    // color used mainly in LMR to hide data that are not in the perimeter
```

Comment:
In codebase, we use `stdcm`. LMR is only for the clients :)
```suggestion
    // color used mainly in stdcm to hide data that are not in the perimeter
```

Reply:
Done